hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -497,21 +497,30 @@ public function upgradeBundle() {
     $this->upgrade_1006();
     $this->upgrade_1008();
     $this->upgrade_1009();
+    $this->upgrade_1010();
```

Comment:
I think some of these might be intentionally omitted, can we remove all new calls in this PR except for the one to your new upgrader?